src/System.Management.Automation/engine/hostifaces/HostUtilities.cs

```diff
@@ -772,10 +772,16 @@ internal static string GetRemotePrompt(RemoteRunspace runspace, string basePromp
             {
                 return basePrompt;
             }
-            else
+            else if (runspace.ConnectionInfo is SSHConnectionInfo)
             {
-                return string.Format(CultureInfo.InvariantCulture, "[{0}]: {1}", runspace.ConnectionInfo.ComputerName, basePrompt);
+                SSHConnectionInfo sshConnectionInfo = runspace.ConnectionInfo as SSHConnectionInfo;
+                if (!string.IsNullOrEmpty(sshConnectionInfo.UserName) &&
+                    !System.Environment.UserName.Equals(sshConnectionInfo.UserName, StringComparison.OrdinalIgnoreCase))
```

Comment:
Seems we could remove `!string.IsNullOrEmpty(sshConnectionInfo.UserName)`.

Reply:
In case where username is not explicitly used, at this point in the code `UserName` is empty so without this check it shows up as `[@computer]` since empty != current username.